components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go

```diff
@@ -51,13 +51,14 @@ func (nc *FakeNamespaceClient) GetEventhubAccessKeys(ctx context.Context, resour
 	}, nc.accessKeysError
 }
 
-func (nc *FakeNamespaceClient) CreateResourceGroup(ctx context.Context, config *azure.Config, name string) (resources.Group, error) {
+func (nc *FakeNamespaceClient) CreateResourceGroup(ctx context.Context, config *azure.Config, name string, tags azure.Tags) (resources.Group, error) {
```

Comment:
Can we test that the tags are passed in a unit test? 🤔

Reply:
Done.